target/linux/generic/hack-6.12/931-mips-ralink-add-missing-include.patch

```diff
@@ -0,0 +1,17 @@
+From b71ab42067204c3d90aa9143b815482b222fee32 Mon Sep 17 00:00:00 2001
+From: Mieczyslaw Nalewaj <namiltd@yahoo.com>
+Date: Mon, 18 Nov 2024 21:28:05 +0100
+Subject: mips: ralink: add missing include
+
```

Comment:
Please also add the commit message here, so it is part of the patch.

Reply:
Done